projects/gloo/cli/pkg/cmd/options/contextoptions/contextoptions.go

```
@@ -35,10 +40,12 @@ func ContextAccessibleFrom(ctx context.Context) (ContextAccessible, error) {
 	return ContextAccessible{}, eris.New("No options set on current context")
 }
 
+// KubecontextFrom pulls the kube context if it was stuffed into the go context.
+// Swallows the no options error that is normally retrieved.
```

if this error is always swallowed, should we just change `ContextAccessibleFrom` to not return an error? this seems to be the only place that func is called

---

Yeah I like that idea